helpers/lib_tasks_docker_release.py

```diff
@@ -390,11 +585,8 @@ def docker_release_dev_image(  # type: ignore
 # /////////////////////////////////////////////////////////////////////////////
 
 
-# TODO(gp): multi_build -> multi_arch
-
-
 @task
-def docker_tag_push_multi_build_local_image_as_dev(  # type: ignore
+def docker_tag_push_multi_arch_local_image_as_dev(  # type: ignore
```

Comment:
Let's not rename it at the moment. Renaming the invoke target would require updating it in all related repositories.
We can handle that in a separate PR.